GSA-TTS/federal-website-standards
# Plug-in the public URL hostname for sitemap

## Context
Search.gov wants the URLs in the sitemap to be fully qualified. We're currently using the Cloud.gov pages sandbox URL in the sitemap, this updates the host to the new public URL so that the sitemap will be correct on launch.

## How to verify this change
**NB**: The sitemap will not update on preview builds because the code only prepends the host name if the branch being built is `main`, this change needs to be verified after it's merged.
1. [Visit the sitemap](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/site/gsa-tts/federal-website-standards/sitemap.xml) and verify that all of the URLs have a `https://digital.standards.gov/` prefix
